src/components/journeyPage/CreateSideBar.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { useState, MouseEvent } from "react";
+import "./CreateSideBar.scss"
+
+export const CreateSideBar: React.FunctionComponent = () => {
+  const [buttonState,setButtonState] = useState(1);
+  const [style, setStyle] = useState({
+    1: "side-bar-button",
+    2: "side-bar-button",
+    3: "side-bar-button",
+    4: "side-bar-button",
+    5: "side-bar-button"
+  });
+
+ function handleButtonState(number:number){
```

Comment:
Can we rename this function to `handleButtonClick`?
And the parameter to `buttonStage` or similar?